### ISSUE TITLE
Can't change notification style

### ISSUE THREAD

**Issue by jmayoff** [cezarsa/silver_bird#405]
I used to have Silver Bird show me desktop notifications. Now they've changed to "notify on page". When I try to change it back and then hit save, it doesn't take and reverts back to "notify on page". Any way to fix this?

EDIT: I'm on Windows 7, with Chrome Version 28.0.1500.72 m

**Comment by j0k3r** [comment]
Same problem here.
I'm also on Windows 7. Chrome version is 29.0.1547.32 beta-m.

**Comment by jmayoff** [comment]
I have since discovered a new plugin, since there are too many problems with this one that take too long to resolve. It's similar, but doesn't have as many options, but has just about everything I need.

https://chrome.google.com/webstore/detail/notifier-for-twitter/ikknnkomiokeodcdkknnhgjmncfiefmn

**Comment by xrstf** [comment]
Will this ever be resolved? The on-page notifications suck ass (broken HTML, looked horrible, unclickable links)...

**Comment by jmayoff** [comment]
Just use the extension I mentioned above. It works.

**Comment by xrstf** [comment]
That's no solution. The Twitter notifier is nice, but is buggy itself (constantly loses my second account, shows an empty timeline, has no "new tweet" indicator color, doesn't allow my to resize the window, ....).

**Comment by j0k3r** [comment]
I use notifier-for-twitter to get proper desktop notification and silverbird to tweet.

**Comment by jmayoff** [comment]
It may not be a perfect solution, but unless you're prepared to dig in to the code yourself, it looks like it's the best solution we've got, at least for the moment.

**Comment by jmlv21104** [comment]
I know its no solution but im a developer and you colud replace the line 185 of /lib/options/page.js with:

if($("#noti_desktop").is(":checked") || true) {

if you do that all your notification should appear in your desktop, then you could add the addon as an unpacked extension.

**Comment by j0k3r** [comment]
The few commits from Aug 29, 2013 fixed this bug.
We still need to load the extension using the dev mode in Chrome, waiting for a new version in the Chrome Store.

**Comment by jmlv21104** [comment]
Yup sorry developer on my own, not part of this project!!!!! Until this guys wont commit via their account i cant do anymore for you guys..... hope it helps at least till they release the new version!
